packages/twenty-front/src/modules/subscription/components/SubscriptionProviderEffect.tsx

```diff
@@ -0,0 +1,65 @@
+import { ON_SUBSCRIPTION_MATCH } from '@/subscription/graphql/subscriptions/onSubscriptionMatch';
+import { useSseClient } from '@/subscription/hooks/useSseClient.util';
+import { subscriptionRegistryState } from '@/subscription/states/subscriptionRegistryState';
+import { print } from 'graphql';
+import { useEffect, useMemo } from 'react';
+import { useRecoilValue } from 'recoil';
+import { isDefined } from 'twenty-shared/utils';
+
+export const SubscriptionProviderEffect = () => {
+  const subscriptionRegistry = useRecoilValue(subscriptionRegistryState);
+
+  const { sseClient } = useSseClient();
+
+  const subscriptions = useMemo(() => {
+    return Array.from(subscriptionRegistry.values()).map((entry) => ({
```

Comment:
~~what is the point of the .map here ?~~ removing the `onRefetch` property. I get it now :)